netcdf_to_gltf_converter/config.py

```diff
@@ -99,6 +99,36 @@ class ModelType(StrEnum):
     XBEACH = "XBEACH"
     """Output from an XBEACH model (regular grid)."""
 
+class ShiftType(StrEnum):
+    """The method to shift the coordinates."""
+    
+    MIN = "min"
```

Comment:
Are there no other shifttypes?

Reply:
No, currently not. 
But this works nicely with pydantic as "min" will be the only accepted string value and it is easy to extend.